src/student_success_tool/analysis/pdp/features/student.py

```diff
@@ -84,3 +89,7 @@ def student_program_of_study_changed_first_year(
     df: pd.DataFrame, *, term_col: str, year_col: str
 ) -> pd.Series:
     return df[term_col].ne(df[year_col]).astype("boolean")
+
+
+def diff_gpa_year_1_to_term_1(df: pd.DataFrame, *, term_col: str, year_col: str) -> pd.Series:
+    return df[year_col] - df[term_col]
```

Comment:
Because these column names are fixed in the real pdp data, you should feel free to provide them as defaults and drop them from the function call:

```suggestion
def diff_gpa_year_1_to_term_1(
    df: pd.DataFrame, *, term_col: str = "gpa_group_term_1", year_col: str = "gpa_group_year_1"
) -> pd.Series:
    return df[year_col] - df[term_col]
```